kinematics_base_test/src/test_kinematics_plugin.cpp

```diff
@@ -27,71 +27,62 @@ const std::string NUM_IK_TESTS = "num_ik_tests";
 const std::string NUM_IK_MULTIPLE_TESTS = "num_ik_multiple_tests";
 const double DEFAULT_SEARCH_DISCRETIZATION = 0.01f;
 
-class KinematicsTest
-{
+class KinematicsTest {
 public:
-
-  bool initialize()
-  {
+  bool initialize() {
     ros::NodeHandle ph("~");
     std::string plugin_name;
 
     // loading plugin
-    kinematics_loader_.reset(new KinematicsLoader("moveit_core", "kinematics::KinematicsBase"));
-    if(ph.getParam(PLUGIN_NAME_PARAM,plugin_name))
-    {
-      try
-      {
-        ROS_INFO_STREAM("Loading "<<plugin_name);
-        kinematics_solver_ = kinematics_loader_->createInstance(plugin_name);
-      }
-      catch(pluginlib::PluginlibException& e)
-      {
-        ROS_ERROR_STREAM("Plugin failed to load: "<<e.what());
+    kinematics_loader_.reset(
+        new KinematicsLoader("moveit_core", "kinematics::KinematicsBase"));
+    if (ph.getParam(PLUGIN_NAME_PARAM, plugin_name)) {
+      try {
+        ROS_INFO_STREAM("Loading " << plugin_name);
+        //        kinematics_solver_ =
+        //        kinematics_loader_->createInstance(plugin_name);
+        kinematics_solver_ = make_shared_ptr(
+            lvalue(kinematics_loader_->createInstance(plugin_name)));
+      } catch (pluginlib::PluginlibException &e) {
+        ROS_ERROR_STREAM("Plugin failed to load: " << e.what());
         EXPECT_TRUE(false);
         return false;
       }
 
-    }
-    else
-    {
-      ROS_ERROR_STREAM("The plugin "<<plugin_name<<" was not found");
+    } else {
+      ROS_ERROR_STREAM("The plugin " << plugin_name << " was not found");
       EXPECT_TRUE(false);
       return false;
     }
 
     // initializing plugin
-    if(ph.getParam(GROUP_PARAM,group_name_) && ph.getParam(TIP_LINK_PARAM,tip_link_) &&
-        ph.getParam(ROOT_LINK_PARAM,root_link_) && ph.getParam(JOINT_NAMES_PARAM,joints_))
-    {
-      if(kinematics_solver_->initialize(ROBOT_DESCRIPTION_PARAM,group_name_,root_link_,tip_link_,DEFAULT_SEARCH_DISCRETIZATION))
-      {
+    if (ph.getParam(GROUP_PARAM, group_name_) &&
+        ph.getParam(TIP_LINK_PARAM, tip_link_) &&
+        ph.getParam(ROOT_LINK_PARAM, root_link_) &&
+        ph.getParam(JOINT_NAMES_PARAM, joints_)) {
+      if (kinematics_solver_->initialize(ROBOT_DESCRIPTION_PARAM, group_name_,
+                                         root_link_, tip_link_,
+                                         DEFAULT_SEARCH_DISCRETIZATION)) {
         ROS_INFO_STREAM("Kinematics Solver plugin initialzed");
-      }
-      else
-      {
+      } else {
```

Comment:
changes like these are a regression of the moveit style guidelines - can you please use the clang-formatter here?